src/nginxconfig/generators/conf/website.conf.js

```diff
@@ -85,6 +85,37 @@ const listenConfig = domain => {
     return httpListen(domain);
 };
 
+
+function httpRedirectConfig(domain, global, domainName, redirectDomain)
+{
```

Comment:
```suggestion
const httpRedirectConfig = (domain, global, domainName, redirectDomain) => {
```
Arrow function, please :)